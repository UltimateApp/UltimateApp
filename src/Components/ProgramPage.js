import React from 'react';
import { StyleSheet, View, Text } from 'react-native';

import theme from '../styles/theme.style';

export const ProgramPage = props => {
  const { navigation, route } = props;
  const { program } = route.params;

  return <View style={styles.programPage} />;
};

export default ProgramPage;

const styles = StyleSheet.create({
  programPage: {
    backgroundColor: theme.BACKGROUND_COLOR_LIGHT,
    height: '100%',
    padding: 20,
  },
});
