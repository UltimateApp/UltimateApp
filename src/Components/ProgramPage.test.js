import React from 'react';
import renderer from 'react-test-renderer';

import fixtures from '../Fixtures/TestFixtures';

import ProgramPage from './ProgramPage';

describe('<ProgramPage />', () => {
  const program = fixtures.programs[0];
  const route = {
    params: {
      program,
    },
  };

  it('renders correctly', () => {
    const tree = renderer.create(<ProgramPage route={route} />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
