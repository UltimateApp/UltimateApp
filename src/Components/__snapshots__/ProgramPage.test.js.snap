// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<ProgramPage /> renders correctly 1`] = `
<View
  style={
    Object {
      "backgroundColor": "#fff",
      "height": "100%",
      "padding": 20,
    }
  }
/>
`;
